.github/workflows/docker-build.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
                     docker run -d --name php-fpm ${{ secrets.DOCKER_USERNAME }}/php-fpm:${DOCKER_PHP_FPM_IMAGE_TAG}
                     docker cp ./ php-fpm:/var/www/html
                     docker exec php-fpm composer install --optimize-autoloader --no-interaction
-                    docker exec php-fpm php phing dirs-create test-dirs-create assets npm build-version-generate
+                    docker exec php-fpm php phing dirs-create test-dirs-create assets npm build-version-generate backend-api-install composer-dev backend-api-oauth-keys-generate
```

Comment:
Maybe a bold suggestion, what do you say to enable frontend api in github actions?

Reply:
added